cmake/SSGCommon.cmake

```diff
@@ -744,8 +739,6 @@ macro(ssg_build_product PRODUCT)
         DESTINATION "${SSG_CONTENT_INSTALL_DIR}")
     install(FILES "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-ds.xml"
         DESTINATION "${SSG_CONTENT_INSTALL_DIR}")
-    install(FILES "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-cpe-dictionary.xml"
```

Comment:
You need to move the installation somewhere. After these changes the CPE dict won't be installed at all.